### ISSUE TITLE
picker selection not correct in Windows high contrast mode

### ISSUE THREAD

**Issue by BrendanMcK** [OfficeDev/office-ui-fabric-react#7811]
### Please provide a reproduction of the bug in a codepen:
 
Visit [PeoplePicker](https://developer.microsoft.com/en-us/fabric#/components/peoplepicker) or [BasePicker](https://developer.microsoft.com/en-us/fabric#/components/pickers) demo pages in Fabric docs, using MSFT Edge, with Windows High Contrast mode enabled.

Type in some text to get the popup, and use keyboard arrows to highlight an item. Also observe colors on mouse hover.

#### Actual behavior:
 
Get inconsistent use of HC colors. Expect that selected items use only the HC selected item text color on HC selected item background colors; but instead get a mix, including non-theme colors.

Example with PeoplePicker:
![image](https://user-images.githubusercontent.com/1754373/51778381-35c8ce80-20b6-11e9-9248-90ff8f42d154.png)

* The top item here is the one with mouse hover: it should be using green-on-white, since that's the selection colors for the HC theme I'm using. Instead, there's some black background creeping in there. (That's likely caused by EDGE's default HC processing; need to use `-ms-high-contrast-adjust: none`  to suppress this - but then have to explicitly state the text/background colors on that and any descendent elements.)

 * The bottom item is the one selected via keyboard. The person's name / job title aren't using the HC theme colors are all here. (Likely caused by use of  `-ms-high-contrast-adjust: none`, but without compensating for the fact that this also turns off default HC processing for the entire subtree in the DOM.)

Example from the Tag Picker:
![image](https://user-images.githubusercontent.com/1754373/51778500-d9b27a00-20b6-11e9-822a-3b15702788cf.png)

Again, top item is showing mouse hover state, bottom is keyboard selection state. Both should be white-on-green.

**Comment by BrendanMcK** [comment]
Sure - I'm using theme "High Contrast #1":

![image](https://user-images.githubusercontent.com/1754373/51952558-cf151f00-23ed-11e9-89c7-b56406171e87.png)

**Comment by dzearing** [comment]
I believe this is a dupe of #4875 which I will reassign to @leddie24 

@Jahnp for fyi

**Comment by Jahnp** [comment]
@dzearing makes sense, thank you!
